src/mmt_models.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from official.modeling import tf_utils
+
+import configs
+import layers as mmt_layers
+import models as mmt_models
+
+
+
+def get_transformer_encoder(mmt_config):
+  """Gets a `MmtEncoder` object.
+  Args:
```

Comment:
Nit: there's usually a blank line between the first line and `Args:`, and also before the `Returns:` section.  (Same for docstrings below too.)  https://google.github.io/styleguide/pyguide.html#383-functions-and-methods